src/shared/components/post/post-listing.tsx

```diff
@@ -321,13 +321,17 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
           {post_view.creator_blocked && (
             <span className="mx-1 badge badge-danger">{"blocked"}</span>
           )}
-          {this.props.showCommunity && (
-            <span>
-              <span class="mx-1"> {i18n.t("to")} </span>
-              <CommunityLink community={post_view.community} />
-            </span>
-          )}
         </li>
+        {this.props.showCommunity && (
+          <li className="list-inline-item">
+            <span class="mx-1"> {i18n.t("to")} </span>
+          </li>
+        )}
+        {this.props.showCommunity && (
```

Comment:
Doesn't need to do this same check twice.

Reply:
I agree. However, the template parser complained if I did something like
```
{this.props.showCommunity && (
  <li>...</li>
  <li>...</li>
)}
```
It said it only accepted one tag as result. Would something like...
```
{this.props.showCommunity && (
  <>
  <li>...</li>
  <li>...</li>
  </>
)}
```
... produce the same output as my repeated check strategy?

I'm new to these frameworks, sorry :/

Reply:
Yep that works.